### ISSUE TITLE
deterministic value ratios

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#679]
if the other or content-to-structure ratio uncertainty is deterministic, we still need a box in which to enter the ratio. No text box exists at the moment.

**Comment by codymccoy** [comment]
My task is to add a text box here and the same for Other Damage tab:
![image](https://user-images.githubusercontent.com/47757848/205755680-1fa00a80-86e9-4e06-aeb1-a2319f5fb0c3.png)

**Comment by rnugent3** [comment]
Yes please - and toss the number from the text box into the occ type builder?

**Comment by codymccoy** [comment]
I am defaulting the value to 0. Is there a better default?

**Comment by codymccoy** [comment]
Any validation on this value? 0 to 1? 0 to 100?

**Comment by rnugent3** [comment]
> 

this looks great.

**Comment by codymccoy** [comment]
Since you want a default of .5 i am guessing that I should validate that the value is between 0 and 1?

**Comment by codymccoy** [comment]
The UI has been updated, error validation is behaving like the other controls (red border, tooltip), the value is saving and loading correctly.
![image](https://user-images.githubusercontent.com/47757848/205992524-a5f92d58-b553-40c6-8bfc-4c4c1a76705a.png)

It has been awhile since I looked over this code. The error validation for all the distribution types in the occtype editor is being ignored when saving. I think this was the decision that we made so that people could at least save their changes even if they were in an error state. I am curious if this ever gets caught when running the compute. The table coordinate values can be in error and each of the uncertainty values can be in error. Triangular can have the min greater than the max for instance. 

I found this comment that I made:
![image](https://user-images.githubusercontent.com/47757848/205993302-dac57269-7908-4d02-b683-2d46eae118dc.png)

It sounds like I left the validation methods that used to get called when the user tried to save in case we wanted to use those to write the messages out to a log file. Any thoughts on this? Seems like we should either write them out to something or delete those unused methods.
